mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -815,4 +829,14 @@ public boolean hasChildren(MailboxPath mailboxPath, MailboxSession session) thro
         Mailbox mailbox = mapper.findMailboxByPath(mailboxPath);
         return mapper.hasChildren(mailbox, session.getPathDelimiter());
     }
+
+    @Override
+    public void setLimitOfAnnotations(int limitOfAnnotations) {
```

Comment:
Is there any runtime command that allows to modify the limit ? If not, please pass by constructor.